### ISSUE TITLE
How to train the model to converge without pre-trained model？ 

### ISSUE THREAD

**Issue by YaohangHan** [ZhengyiLuo/KeypointNet#1]
I run the code on GTX TITAN (6GB) with batch_size=8 ,and the loss converge around 0.1. But the predict result on test images are terrible, I can not sure whether the batch size too small to lead to a good result.

**Comment by ZhengyiLuo** [comment]
Which loss are you referring to? Also, this is a forked repo from: [](https://github.com/tensorflow/models/tree/master/research/keypointnet), which may provide more information.
